server/views/components/instagram-thumbnail/index.njk

```diff
@@ -0,0 +1,31 @@
+<a href="{{ model.url }}" class="instagram-thumbnail"
+  data-track-click='{"screenName": "{{ model.screenName }}", "createdAt": "{{ model.createdAt }}", "postId": {{ model.id }}}'>
```

Comment:
Analytics for analytics sake is something GA is really good at.
Mixpanel is really good at hyper focussed analytics.

Maybe speak to @hthair @jennpb and @Heesoomoon about what they'd like?